### ISSUE TITLE
RemovedInDjango40Warning: ugettext_lazy() is deprecated

### ISSUE THREAD

**Issue by deeprave** [jazzband/wagtailmenus#382]
I am getting the following warning for each use of _(...) when used with Django 3.1.3, wagtail 2.11:

**RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy()**

An easy and obvious fix, PR incoming shortly.

**Comment by Crocmagnon** [comment]
This  is becoming more urgent since wagtailmenus now doesn't work with the latest setup of wagtail 2.16 (using Django 4 by default)

**Comment by DanielSwain** [comment]
Agreeing with @Crocmagnon.  I tried this package in a project and ended up not making use of it.  When I tried to remove the package form the venv and from `INSTALLED_APPS`, I got an error that one of the project's migrations is dependent on one of the `wagtailmenus` migrations (0022).  Since I can't remove `wagtailmenus` from `INSTALLED_APPS` because of this migrations dependency, I now have to (artificially) constrain this project to Django 3 when I would otherwise have just upgraded it to 4.
